Cargo.toml

```diff
@@ -18,3 +18,9 @@ rustyline = "^8.0.0"
 
 [features]
 use_bindgen = ["drm-ffi/use_bindgen"]
+
+[workspace]
+members = [
+    "drm-ffi",
+    "drm-ffi/drm-sys",
+]
```

Comment:
Sure! Seems I forgot to add this.

Reply:
I was surprised to see a bunch of warnings in my editor, and it took a while to notice! `cargo clippy --all` on Rust 1.51 seems to lint these crates regardless of the workspace.